### PR TITLE
psr4 improvements for the code base

### PR DESCRIPTION
**Pull request by olalekan-agbaje**
The current repo is currently on psr0 and makes it difficult to manage with PHP frameworks.
These changes move the code to psr4 and adds namespaces.

### CODE REVIEW

**Review by ayinloya — COMMENTED** [review]
This looks good
We'll need a changelog entry

File: examples/biometric_kyc.php [hunk 1]
```diff
@@ -1,8 +1,12 @@
 <?php
 
+use SmileIdentity\JobType;
+use SmileIdentity\ImageType;
+use SmileIdentity\SmileIdentityCore;
+
 // Autoload the dependencies
 require 'vendor/autoload.php';
-include 'SmileIdentityCore.php';
+// include 'SmileIdentityCore.php';
```
Comment (ayinloya) [inline examples/biometric_kyc.php:9]: we can remove this

File: examples/business_verification.php [hunk 2]
```diff
@@ -1,8 +1,11 @@
 <?php
 
+use SmileIdentity\JobType;
+use SmileIdentity\SmileIdentityCore;
+
 // Autoload the dependencies
 require 'vendor/autoload.php';
-include 'SmileIdentityCore.php';
+// include 'SmileIdentityCore.php';
```
Comment (ayinloya) [inline examples/business_verification.php:8]:
```suggestion
```

File: examples/smart_selfie_authentication.php [hunk 3]
```diff
@@ -1,8 +1,12 @@
 <?php
 
+use SmileIdentity\JobType;
+use SmileIdentity\ImageType;
+use SmileIdentity\SmileIdentityCore;
+
 // Autoload the dependencies
 require 'vendor/autoload.php';
-include 'SmileIdentityCore.php';
+// include 'SmileIdentityCore.php';
```
Comment (ayinloya) [inline examples/smart_selfie_authentication.php:9]:
```suggestion
```

File: lib/IdApi.php [hunk 4]
```diff
@@ -1,15 +1,17 @@
 <?php
-spl_autoload_register(function ($class) {
-    require_once($class . '.php');
-});
+
+namespace SmileIdentity;
+// spl_autoload_register(function ($class) {
+//     require_once($class . '.php');
+// });
```
Comment (ayinloya) [inline lib/IdApi.php:6]:
```suggestion
```

**Review by nii-mants3 — APPROVED** [review]
LGTM 👍

**Review by ayinloya — APPROVED** [review]

**Pull request merged**